provider/datagen/src/transform/cldr/datetime/symbols.rs

```diff
@@ -86,64 +86,109 @@ fn get_month_code_map(calendar: &str) -> &'static [TinyStr4] {
+                    .into_iter()
+                    .collect(),
+            ),
+            ("chinese".to_string(), [].into_iter().collect()),
+            ("japanese".to_string(), japanese.clone()),
+            ("japanext".to_string(), japanese),
+            (
+                "coptic".to_string(),
+                [
+                    // Before Diocletian
+                    ("0".to_string(), tinystr!(16, "bd")),
+                    // Anno Diocletian/After Diocletian
+                    ("1".to_string(), tinystr!(16, "ad")),
+                ]
+                .into_iter()
+                .collect(),
+            ),
+            ("dangi".to_string(), [].into_iter().collect()),
+            (
+                "indian".to_string(),
+                [("0".to_string(), tinystr!(16, "saka"))]
+                    .into_iter()
+                    .collect(),
+            ),
+            (
+                "islamic".to_string(),
+                [("0".to_string(), tinystr!(16, "islamic"))]
+                    .into_iter()
+                    .collect(),
+            ),
+            (
+                "islamicc".to_string(),
+                [("0".to_string(), tinystr!(16, "islamic"))]
+                    .into_iter()
+                    .collect(),
+            ),
+            (
+                "umalqura".to_string(),
+                [("0".to_string(), tinystr!(16, "islamic"))]
+                    .into_iter()
+                    .collect(),
+            ),
+            (
+                "tbla".to_string(),
+                [("0".to_string(), tinystr!(16, "islamic"))]
+                    .into_iter()
+                    .collect(),
+            ),
+            (
+                "persian".to_string(),
+                [("0".to_string(), tinystr!(16, "ah"))]
+                    .into_iter()
+                    .collect(),
+            ),
+            (
+                "hebrew".to_string(),
+                [("0".to_string(), tinystr!(16, "hebrew"))]
+                    .into_iter()
+                    .collect(),
+            ),
+            (
+                "ethiopic".to_string(),
+                [
+                    ("0".to_string(), tinystr!(16, "incar")),
+                    ("1".to_string(), tinystr!(16, "pre-incar")),
+                    ("2".to_string(), tinystr!(16, "mundi")),
+                ]
+                .into_iter()
+                .collect(),
+            ),
+            (
+                "roc".to_string(),
+                [
+                    ("0".to_string(), tinystr!(16, "roc-inverse")),
+                    ("1".to_string(), tinystr!(16, "roc")),
+                ]
+                .into_iter()
+                .collect(),
+            ),
         ]
         .into_iter()
-        .collect(),
-        _ => panic!("Era map unknown for {calendar}"),
-    }
+        .collect()
+    });
+    map.get(calendar)
+        .unwrap_or_else(|| panic!("Unknown calendar {}", calendar))
```

Comment:
Nit: Keep the error name "Era map unknown for {calendar}" to make it more greppable